src/Controller/Controller.php

```diff
@@ -159,7 +163,7 @@ class Controller implements EventListenerInterface, EventDispatcherInterface
      *
      * @var bool
      */
-    public $autoRender = true;
+    protected $autoRender = true;
```

Comment:
They could. But making just _this_ property private would introduce inconsistencies with other parts of the framework and break compatibility.